src/smart-components/group/groups.js

```diff
@@ -30,16 +30,23 @@ const Groups = () => {
   const [ removeGroupsList, setRemoveGroupsList ] = useState([]);
 
   const dispatch = useDispatch();
-  const { groups, pagination, userIdentity, isLoading } = useSelector(({ groupReducer: { groups, isLoading }}) => ({
-    groups: groups.data,
+  const { groups, pagination, userIdentity, isLoading } = useSelector(({ groupReducer: { groups, isLoading, systemGroup }}) => ({
+    groups: groups.data && systemGroup ?
```

Comment:
I don't think you need to perform this check
```JS
groups: [
  ...(systemGroup?.name?.toLowerCase()?.includes(filterValue) ? [ systemGroup ] : []),
  ...groups?.data?.filter(({ platform_default } = {}) => !platform_default) || []
]
```

Reply:
@mmenestr I don't remeber what we agreed on, do we want to show or hide default group when filtering? This will hide it, which is totally fine if we want to do that. Just wanted to be sure about it.

Reply:
@karelhala @mmenestr current implementations shows the default group on filter only if it matches given filterValue. That's what I thought to be expected.